mne/viz/tests/test_raw.py

```diff
@@ -308,7 +295,7 @@ def test_plot_raw_ssp_interaction(raw):
     fig = raw.plot()
     # open SSP window
     _fake_click(fig, fig.mne.ax_proj, [0.5, 0.5])
-    assert len(plt.get_fignums()) == 2
+    assert len(plt.get_fignums()) == 2  # XXX this fails with minimal deps
```

Comment:
revert, doesn't seem to fail?

Reply:
I had deleted that once, must have snuck back in during one of the many rebases